apps/fluent-tester/src/RNTester/TestComponents/Avatar/AvatarTest.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import { Avatar } from '@fluentui/react-native';
+import { Stack } from '@fluentui-react-native/stack';
+import { stackStyle } from '../Common/styles';
+import { AVATAR_TESTPAGE } from './consts';
+import { Test, TestSection, PlatformStatus } from '../Test';
+
+const avatar: React.FunctionComponent<{}> = () => {
+  return (
+    <Stack style={stackStyle}>
+      <Avatar avatarData={{ primaryText: 'John Smith' }} />
+      <Avatar avatarData={{ primaryText: 'John Smith' }} avatarStyle="square" />
+      <Avatar avatarData={{ primaryText: 'John Smith', presence: 'available' }} />
+      <Avatar avatarData={{ primaryText: 'John Smith', secondaryText: 'johnsmith@microsoft.com', presence: 'outOfOffice' }} />
+      <Avatar avatarData={{ secondaryText: 'johnsmith@microsoft.com', presence: 'outOfOffice' }} />
```

Comment:
As @JasonVMo has already pointed out, having a sub-object like so is not the best/efficient way to do this. The biggest TODO of this PR that stops it from being a full PR is having a flat list of props on the JSX "Avatar" component, which then maps to this object when passed to the inner native component

https://github.com/microsoft/fluentui-react-native/pull/372#discussion_r474326015

Reply:
Perhaps we can use the memo cache as mentioned here (https://github.com/microsoft/fluentui-react-native/pull/462#discussion_r492195973) to map the props and make sure we don't rerender every time

Reply:
I think this usage shows some of the issues with the interface. From the perspective of the user, most of the data they want to input is actually in the sub-object, which is really a useless level of indirection for 1, but also has the risk of this style of usage with inline objects being specified. There are linting rules for RN that will detect inline styles but I don't think they work for other objects.

Similar to shimmer, it's fine for the native component to have an interface like this, the JS side can translate from a good JS interface to the native one.

Reply:
Also internally you can memo on the input object reference, but that breaks if people use it like you specified above. Alternatively you could split apart the avatarData into the separate props, then memo on the key combinations to form the internal object to pass to native, which is fine. But at that point what value did the avatarData prop have at the JS level if it has to be broken apart.